lib/rubocop/cop/rails/reversible_migration.rb

```diff
@@ -179,7 +179,7 @@ class ReversibleMigration < Base
         MSG = '%<action>s is not reversible.'
 
         def_node_matcher :irreversible_schema_statement_call, <<~PATTERN
-          (send nil? ${:execute :remove_belongs_to} ...)
+          (send nil? ${:execute :remove_belongs_to :change_column} ...)
```

Comment:
Unrelated to the PR.
Do you mind adding `remove_belongs_to`'s alias `remove_belongs_to` to this list along the way, say in a separate commit?

Reply:
Sure.  I believe you mean `remove_reference` :)
https://github.com/rails/rails/blob/b6bc552c1f566ef052c6791be890148ccbd9b226/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb#L1023

Reply:
Added separate commit